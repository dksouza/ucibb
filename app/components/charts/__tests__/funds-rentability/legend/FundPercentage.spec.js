import React from 'react';
import { shallow } from 'enzyme';
import FundPercentage from '../../../funds-rentability/legend/fundPercentage';

let wrapper = {};
let props = {};

const name = 'Fundo';
const percent = 10;

describe('FundPercentage', () => {
  beforeAll(() => {
    props = {
      name,
      percent,
    };

    return props;
  });

  beforeEach(() => {
    wrapper = shallow(<FundPercentage {...props} />);
    return wrapper;
  });

  it('Shallow View', () => {
    expect(wrapper.find('View').exists()).toBe(true);
  });

  it('Shallow BBText', () => {
    expect(wrapper.find('BBText').exists()).toBe(true);
  });

  it('Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
