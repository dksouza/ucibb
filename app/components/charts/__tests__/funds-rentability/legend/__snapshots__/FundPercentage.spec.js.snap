// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FundPercentage Snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <BBText
    accessibilityLabel="Porcentagem que esse fundo ocupa no gráfico é de
 10 porcentos"
    regular={true}
    small={true}
    style={
      Object {
        "color": "rgba(0,0,0,0.54)",
        "paddingTop": 0,
      }
    }
  >
    10
    %
  </BBText>
  <BBText
    accessibilityLabel="Nome do fundo: 10"
    small={true}
    style={
      Object {
        "marginLeft": 4,
        "paddingTop": 0,
      }
    }
  >
    Fundo
  </BBText>
</View>
`;
